keylime/cloud_verifier_tornado.py

```diff
@@ -745,6 +745,15 @@ def delete(self):
             web_util.echo_json_response(self, 500, "Failed to get allowlist")
             raise
 
+        try:
+            agent = session.query(VerfierMain).filter_by(ima_policy_id=ima_policy.id).one_or_none()
+        except SQLAlchemyError as e:
+            logger.error("SQLAlchemy Error: %s", e)
+            raise
+        if agent is not None:
+            web_util.echo_json_response(self, 409, f"Allowlist in use by agent {agent.agent_id}")
```

Comment:
I would make this more descriptive. You've already provided the `409` code, so you can make the error message user-friendly. Something like "Can't delete allowlist as it's currently being used by agent ###".